src/utils.c

```diff
@@ -1788,6 +1788,9 @@ void utils_tidy_path(gchar *filename)
 	const gchar *needle;
 	gboolean preserve_double_backslash = FALSE;
 
+	if (utils_is_uri(filename))
+		return;
```

Comment:
>@techee did you notice the release date THIS MONTH

Sure, I'm aware of the release and the patches are definitely not meant for it. It's just that I had time now and the bigger pull requests are better to be merged early in the next release cycle to get better testing (of course if Colomban has time to review them).